diffkemp/simpll/SourceCodeUtils.cpp

```diff
@@ -103,8 +129,25 @@ std::unordered_map<std::string, MacroElement> getAllMacrosOnLine(
     return usedMacroMap;
 }
 
+/// Takes a list of parameter-argument pairs and expand them on places where
+/// are a part of a composite macro name joined by ##.
+void expandCompositeMacroNames(std::vector<std::pair<std::string, std::string>>
+        args, std::string &body) {
```

Comment:
Nicer alignment of the header:

    void expandCompositeMacroNames(
        std::vector<std::pair<std::string, std::string>> args, 
        std::string &body) {